app/parse.py

```diff
@@ -1,9 +1,22 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, astuple, fields
 from urllib.parse import urljoin
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.common import NoSuchElementException
+import csv
 
 
 BASE_URL = "https://webscraper.io/"
 HOME_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/")
+PRODUCTS_URLS = {
+    "home": HOME_URL,
+    "computers": urljoin(HOME_URL, "computers"),
+    "laptops": urljoin(HOME_URL, "computers/laptops"),
+    "tablets": urljoin(HOME_URL, "computers/tablets"),
+    "phones": urljoin(HOME_URL, "phones"),
+    "touch": urljoin(HOME_URL, "phones/touch")
```

Comment:
/

Reply:
I don't understand your comment. I think that the URL structure is correct. Should I write '/' at the end of each link or at the beginning of each link?